contracts/KyberPoolMaster.sol

```diff
@@ -355,6 +359,85 @@ contract KyberPoolMaster is Ownable {
         );
     }
 
+    /**
+     * @dev  Queries the amount of unclaimed rewards for the pool member
+     *       return 0 if PoolMaster has not called claimRewardMaster
+     *       return 0 if PoolMember has previously claimed reward for the epoch
+     *       return 0 if PoolMember has not stake for the epoch
+     *       return 0 if PoolMember has not delegated it stake to this contract for the epoch
+     * @param epoch for which epoch the memmber is querying unclaimed reward
+     */
+    function getUnclaimedRewardsMember(uint256 epoch)
+        public
+        view
+        returns (uint256)
+    {
+        if (!claimedPoolReward[epoch]) {
+            return 0;
+        }
+
+        address poolMember = msg.sender;
+
+        if (claimedDelegateReward[epoch][poolMember]) {
+            return 0;
+        }
+
+        (
+            uint256 stake,
+            uint256 delegatedStake,
+            address delegatedAddr
+        ) = kyberStaking.getStakerDataForPastEpoch(poolMember, epoch);
+
+        if (stake == 0) {
+            return 0;
+        }
+
+        if (delegatedAddr != address(this)) {
+            return 0;
+        }
+
+        Reward memory rewardForEpoch = memberRewards[epoch];
+
+        return
+            calculateRewardsShare(
+                stake,
+                rewardForEpoch.totalStaked,
+                rewardForEpoch.totalRewards
+            );
+    }
+
+    /**
+     * @dev Claims reward for poolMember has not claimed for an epoch previously and the poolMaster has claimed rewards for the pool
```

Comment:
Let's also document in this method's `@dev` description that the contract will keep locked remainings from rounding at a wei level.